pkg/windows/windows.go

```diff
@@ -703,3 +785,8 @@ func (vm *windows) newFileInfo(path string) (*payload.FileInfo, error) {
 func mkdirCmd(dirName string) string {
 	return "if not exist " + dirName + " mkdir " + dirName
 }
+
+// rmDirCmd returns the Windows command to recursively remove a directory if it exists
+func rmDirCmd(dirName string) string {
+	return "if exist " + dirName + " rmdir " + dirName + " /s /q"
```

Comment:
Are we pushing to standardize Powershell usage over interacting with command prompt?

Reply:
There I'm going with the pattern set by the existing mkDirCmd function. I've just created https://issues.redhat.com/browse/WINC-636